### PR TITLE
Fixes #25710 - Notify when scap feature missing

### PR DESCRIPTION
**Pull request by upadhyeammit**
This adds error as notification to handle condition when there is 'OpenSCAP proxy' but its actually not having Openscap feature, so user will be notified and can unset or correct the 'OpenSCAP proxy'.

### CODE REVIEW

File: app/controllers/concerns/foreman_openscap/hosts_and_hostgroups_extensions.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module ForemanOpenscap
+  module HostsAndHostgroupsExtensions
```
Comment (xprazak2) [inline app/controllers/concerns/foreman_openscap/hosts_and_hostgroups_extensions.rb]: The module name suggests that it extends hosts and hostgroups, but it extends the controllers, so it should be `HostsAndHostgroupsControllerExtensions`.

**Comment by xprazak2** [comment]
Works! I left just one comment inline.

File: app/controllers/concerns/foreman_openscap/hosts_and_hostgroups_extensions.rb (hunk 1 above)
Comment (upadhyeammit) [reply app/controllers/concerns/foreman_openscap/hosts_and_hostgroups_extensions.rb]: Fixed!

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**